modules/stanford_basic_page_types/templates/components/basic_page_types_list/basic-page-types-list.twig

```diff
@@ -0,0 +1,52 @@
+{#
+/**
+ * @file
+ * Basic Page Types List
+ *
+ * Available variables:
+ * - attributes: For additional HTML attributes not already provided.
+ * - basic_page_types_category: The taxonomy term for the type.
+ * - basic_page_types_title: The Page title.
+ * - basic_page_types_description: The page description.
+ * - basic_page_types_page_image: the page image.
+ * - basic_page_types_footer: an open footer.
+ */
+#}
+{%- set attributes = attributes.addClass(['su-basic-page-types-list', modifier_class]) -%}
+
+<ul{{ attributes }}>
+
+{%- block block_basic_page_types_category  %}
+{% if basic_page_types_category|render|striptags('<drupal-render-placeholder>') is not empty %}
```

Comment:
`|render|striptags('<drupal-render-placeholder>')` can be accomplished with `|render_clean`

Reply:
This is fixed.